core/templates/core/index.html

```diff
@@ -3,21 +3,36 @@
 {% block title %}Strona Główna{% endblock %}
 
 {% block content %}
+    <nav class="bg-white border-gray-200 dark:bg-gray-900">
+        <div class="flex flex-wrap justify-between items-center mx-auto max-w-screen-xl p-4">
+            <a href="https://github.com/Gwardia-Czapli/GwardiaHub" class="hover:text-blue-100 text-gray-400 transition-colors duration-300" target="_blank">
```

Comment:
I think you missunderstood, it was supposed to be link to gwardia module, so you should probably do 
```suggestion
            <a href="{% url 'gwardia' %}" class="hover:text-blue-100 text-gray-400 transition-colors duration-300" target="_blank">
```
We don't need image to it, so leave just text

Reply:
uhh this is what shows up after i changed it, no idea what it means
![image](https://github.com/Gwardia-Czapli/GwardiaHub/assets/162600030/edaf23c1-f569-4685-859f-c355ddde5c1d)

Reply:
Welp, we don't have gwardia in urls patterns, because it's not created yet, but don;t worry about that, person who will create gwardia app should add it to urls patterns

Reply:
okay, ty!!